verl/workers/rollout/vllm_rollout/vllm_rollout.py

```diff
@@ -184,6 +185,14 @@ def generate_sequences(self, prompts: DataProto, **kwargs) -> DataProto:
                 'temperature': 0,
                 'n': 1  # if greedy, only 1 response
             }
+        elif is_validate and do_sample:
```

Comment:
`and do_sample` is redundant. But leaving it as comment might making this branch more readable.


```suggestion
        elif is_validate: # and do_sample
```

Reply:
This `do_sample` is necessary. If the user accidentally set do_sample=False while setting `temperature>0`. Then, sampling parameters will be override

Reply:
I mean there is a `if not do_sample` above, so here `elif` implicitly implies `and do_sample`.

Reply:
Yes, that's what I refer to. When do_sample=False and validate=True. Without a do_sample check, the sampling parameters would be wrong if someone accidentally set temperature>0

Reply:
I see, we should use if not elif...